.github/workflows/build_and_test.yml

```diff
@@ -199,6 +199,23 @@ jobs:
       build-tools-image: ${{ needs.build-build-tools-image.outputs.image }}-bookworm
     secrets: inherit
 
+  validate-compute-manifest:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
```

Comment:
Node 20 seems a pretty old one, the latest version is 24, the latest LTS is 22, I would use one of them